cmd/update/project.go

```diff
@@ -88,8 +88,9 @@ func updateProjectsFunc(ctx context.Context, args []string, cmdCtx cmdCore.Comma
 	if err != nil {
 		return err
 	}
-	if projectSpec.Id == "" {
-		return fmt.Errorf(clierrors.ErrProjectNotPassed)
+
+	if projectSpec.Name == "" {
```

Comment:
I think this isn't quite correct, since we do need the id to be specified to uniquely identify the project to update. However, we might not necessary want to update the name (this is just the human readable string) since the UpdateProject call does only updates modified fields on the backend.

I think what we should do is check that if the projectSpec.Id is empty then read it from the global flytectl `-p` project flag and if that's omitted, raise an error.

Reply:
Ah, you're right. I thought `-p` is passed to `projectSpec.name`. Updating